### PR TITLE
MobileServices documentation cleanup

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

**Review by kevinlind — COMMENTED** [review]
I believe we should have the `AEP 3.x` tabs before the `ACP 2.x` tabs, like we do for MobileCore. https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference
Also, can you update the tabs/examples in the README.md? In the README.md you may also need to include missing code examples as it appears the ACP 2.x example only have ObjC code and the AEP 3.x examples only have Swift code. https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics-mobile-services

**Comment by emdobrin** [comment]
In addition to Kevin's comment, can we also rename the API sections by API name in alphabetical order on the API Reference page? See Analytics as an example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference

**Comment by yangyansong-adbe** [comment]
> In addition to Kevin's comment, can we also rename the API sections by API name in alphabetical order on the API Reference page? See Analytics as an example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference

@emdobrin  If it's the convention for SDK documentation, can you update the "documentation guidance" (https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=adms&title=SDK+Documentation#SDKDocumentation-DocumentationupdatesforSwiftAEPversion)? Then we can make consistency for all API docs.

**Comment by yangyansong-adbe** [comment]
@kevinlind @emdobrin your comments have addressed. Can you give another look?

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 1]
```diff
@@ -1,56 +1,120 @@
 # Mobile Services API reference
 
-## Version of the Mobile Services extension
+## Process referrer intent
```
Comment (kevinlind) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:3]: Rename title to API name `processReferrer`

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 2]
```diff
@@ -1,56 +1,120 @@
...
+**Java**
+
+```java
+public  void onReceive(Context context, Intent intent) {
+    MobileServices.processReferrer(context, intent);
+}
 ```
 
-### Swift
+## Process the Google Play Install Referrer
```
Comment (kevinlind) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:29]: Rename title to name of API `processGooglePlayInstallReferrerUrl` and move section above `processReferrer` for alphabetical listing of APIs.

File: using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md [hunk 3]
```diff
@@ -72,169 +136,112 @@ Uri testUri = new Uri.Builder()
...
-You can use this API to process the referrer intent that was received from Android.
+{% endtab %}
 
-{% hint style="warning" %}
-This API is **only** available in Android.
-{% endhint %}
+{% endtabs %}
 
-### processReferrer
+## extensionVersion
```
Comment (kevinlind) [inline using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:198]: Move `extensionVersion` to the top for alphabetical listing of APIs.

Comment (yangyansong-adbe) [reply using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md:198]: https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#extensionversion
We didn't do that for Analytics documentation, so I think we'd better not to change it.

**Pull request merged**